tests/Mocks/ERC165Interfaces.test.ts

```diff
@@ -59,6 +59,11 @@ describe("Calculate LSP interfaces", () => {
     expect(result).to.equal(INTERFACE_IDS.LSP9Vault);
   });
 
+  it("LSP11", async () => {
+    const result = await contract.callStatic.calculateInterfaceLSP11();
```

Comment:
`.callStatic` is not needed since it is a `pure` function and it does not create a transaction.

```suggestion
    const result = await contract.calculateInterfaceLSP11();
```